notes/2020-05-13.md

```diff
@@ -0,0 +1,46 @@
+# Stencil Contributors Chat - May 13, 2020
+
+## SSG
+
+* Now documented. Everything is supported and tested out. Also includes types
+* Has a separate config file. Helps keep things separate, and is opt-in
+* https://stenciljs.com/docs/static-site-generation
+
+## Debugging
+
+* Now possible with a single file, can loaded and testing things out in VS Code
```

Comment:
Is there a docs page or PR this could link to? I heard this mentioned on the call, but missed the specifics. Sounds exciting.

Reply:
@adamdbradley can probably speak to this